### PR TITLE
Separated introduction section of MXD tutorial

### PR DESCRIPTION
**Pull request by jkbquabeck**
## Description
 
This PR publishes the first step of the reworked MXD tutorial. This was requested in issue #315 (https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/issues/315). The MXD tutorials are to be published in different repositories, starting with tutorial_resources.

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [ ] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

File: mxd/1_MXD_Introduction.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# 1 Introduction
+
+With the Minimum Tractus-X Dataspace, IT departments can set up their own little data space to perform a data exchange between two parties (Bob and Alice in our example). The MXD can be used as a sandbox for testing.
```
Comment (paullatzelsperger) [inline mxd/1_MXD_Introduction.md:3]:
```suggestion
With the Minimum Tractus-X Dataspace, IT departments can set up their own data space to perform a data exchange between two parties (Bob and Alice in our example). The MXD can be used as a sandbox for testing.
```

Comment (paullatzelsperger) [inline mxd/1_MXD_Introduction.md:3]:
```suggestion
With the Minimum Tractus-X Dataspace (MXD), IT departments can set up their own little data space to perform a data exchange between two parties (Bob and Alice in our example). The MXD can be used as a sandbox for testing.
```

File: mxd/1_MXD_Introduction.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+For whom is that relevant?
```
Comment (paullatzelsperger) [inline mxd/1_MXD_Introduction.md:5]:
```suggestion
## 0. Intended audience
```

File: mxd/1_MXD_Introduction.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+This tutorial is designed for companies that want to perform data exchange in a "real" Catena-X data space infrastructure.
```
Comment (paullatzelsperger) [inline mxd/1_MXD_Introduction.md:6]: That is not quite true.
```suggestion
This tutorial is designed for developers who want to get their hands dirty, for companies that want to perform data exchange in actual Catena-X data space infrastructure and generally for curious minds who want to explore dataspaces.
```

File: mxd/1_MXD_Introduction.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+Who should execute the tutorial?
```
Comment (paullatzelsperger) [inline mxd/1_MXD_Introduction.md:8]: move to intended audience. And you don't "execute" a tutorial. You "read" it, or fwiw you "do" it.

File: mxd/1_MXD_Introduction.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+IT-Employees with the following skills and previous experience:
```
Comment (paullatzelsperger) [inline mxd/1_MXD_Introduction.md:9]:
```suggestion
Required knowledge/skills:
```

File: mxd/1_MXD_Introduction.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+- Beginner level docker and kubernetes
+- Beginner level terraform
+- Basic linux system commands
+
+## 1.1 Components & Architecture
+
+By performing this tutorial a data space will be set up including the following components:
+
+- 2 EDC Connectors (Called Bob and Alice)
+- 1 Managed Identity Wallet
```
Comment (paullatzelsperger) [inline mxd/1_MXD_Introduction.md:20]: explain briefly what these components are. people may not know

File: mxd/1_MXD_Introduction.md [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+- 1 Keycloak instance
+- 1 Postgres data base
+
+## 1.2 Prerequisites
+
+In order to run the Minimum Tractus-X Dataspace "MXD" on your local machine, please make sure the following
```
Comment (paullatzelsperger) [inline mxd/1_MXD_Introduction.md:26]:
```suggestion
In order to run the Minimum Tractus-X Dataspace on your local machine, please make sure the following
```

**Review by paullatzelsperger — CHANGES_REQUESTED** [review]

**Comment by jkbquabeck** [comment]
@paullatzelsperger adapted your suggestions.
in case you have not spoken to @stephanbcbauer: we will require your approval for the contents of this PR, but upload these on the tutorial branch of the github.io repository.
We will update the MXD tutorial here after testing of the updated setup etc. has been concluded. The point is to keep this document lean for everyone just interested in setting up the MXD.

**Review by paullatzelsperger — APPROVED** [review]

**Pull request merged**